recipes/sylph/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -euo
+
+# Add workaround for SSH-based Git connections from Rust/cargo.  See https://github.com/rust-lang/cargo/issues/2078 for details.
+# We set CARGO_HOME because we don't pass on HOME to conda-build, thus rendering the default "${HOME}/.cargo" defunct.
+export CARGO_NET_GIT_FETCH_WITH_CLI=true CARGO_HOME="$(pwd)/.cargo"
+
+# build statically linked binary with Rust
+RUST_BACKTRACE=1 cargo install --verbose --path . --root $PREFIX
```

Comment:
Can you try pulling in the rust packages through dependencies in the `run` section of the meta.yaml instead of installing them with `cargo` in the build script? The build process itself should not install any dependencies from somewhere else.

Reply:
I can try, but I'm not sure the best way to go about this. I'm not the most experienced software eng, and please correct my logic if I say something silly, but:

1. Many other rust-only libraries have `run` empty: see https://github.com/bioconda/bioconda-recipes/blob/7f0b5aee83da3984ab18cb7f2ae1479c0f8df838/recipes/fqgrep/meta.yaml or https://github.com/bioconda/bioconda-recipes/blob/7f0b5aee83da3984ab18cb7f2ae1479c0f8df838/recipes/splitp/meta.yaml from https://github.com/fulcrumgenomics/ and https://github.com/COMBINE-lab/ respectively. 
2. My understanding is that the dependencies specified in `cargo` are libraries that are downloaded, compiled, and linked during the `cargo install` command. As such, these are compile-time dependencies, and not run-time dependencies. This building procedure is the reason why gcc, make, and cmake, etc have to be included as requirements, but the end result is that these dependencies are meant to be handled by cargo and not externally.

Reply:
Rust packages are statically compiled so this isn't needed.